include/zim/archive.h

```diff
@@ -313,6 +313,22 @@ namespace zim
        */
       bool is_multiPart() const;
 
+      /** Get if the zim archive uses the new namespace scheme.
+       *
+       * Recent zim file use the new namespace scheme.
+       *
+       * On user perspective, it means that :
+       * - On old namespace scheme :
+       *  . All entries are accessible, either using `getEntryByPath` with a specific namespace
+       *    or simply iterating over the entries (with `iter*` methods).
+       *  . Entry's path has namespace included ("A/foo.html")
+       * - On new namespace scheme :
+       *  . Only the "user" entries are accessible with `getEntryByPath` and `iter*` methods.
+       *    To access metadatas, use `getMetadata` method.
+       *  . Entry's path do not contains namespace ("foo.html")
+       */
+      bool useNewNamespaceScheme() const;
```

Comment:
As a query method it better be named `usesNewNamespaceScheme()`

Reply:
Hum. You are technically right. But it maybe because I'm not a native English speaker but I'm pretty sure I will miss this `s`.

Maybe we could use the more standard way to query things with `is` or `has` ?

Reply:
`hasNewNamespaceScheme()` is fine